bindings/py/tests/sdr/Metrics_test.py

```diff
@@ -138,8 +138,8 @@ def testMetricsExample(self):
         assert( type(M.overlap)             == sdr.Overlap)
         gold = """SDR( 2000 )
     Sparsity Min/Mean/Std/Max 0.1 / 0.0999989 / 5.20038e-06 / 0.1
-    Activation Frequency Min/Mean/Std/Max 0 / 0.100001 / 0.0974391 / 0.619048
-    Entropy 0.830798
+    Activation Frequency Min/Mean/Std/Max 0 / 0.100001 / 0.095393 / 0.571429
+    Entropy 0.834509
```

Comment:
This test for exact output changed because method `sdr.addNoise()` depends on the order of data in `sdr.sparse`, which is now sorted.